build-rev.sh

```diff
@@ -19,6 +19,19 @@ GIT_COMMIT=$(git rev-parse HEAD)
 GIT_TAG=$(git describe --tags --dirty --match=v[0-9]* 2> /dev/null || git rev-parse --short HEAD;)
 GIT_UTCTIME=$(date -u '+%Y/%m/%d %H:%M:%S')
 
+# Cross-compilation occurs when you set TARGET_TRIPLE to something other than
```

Comment:
This comment doesn't make any sense in this context. I see you've copy-pasted it from the `Makefile`. You can just delete it.

Reply:
Yeap I just want to avoid that, if `Makefile` disappear, this comments does not go away with it. But yes does not make much sense there. Let me remove it.